### PR TITLE
IGVF-817-in-vitro-system-UI

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: components/common-data-items.js [hunk 1]
```diff
@@ -151,6 +151,21 @@ export function SampleDataItems({
           </DataItemValue>
         </>
       )}
+      {truthyOrZero(item.sorted_fraction) && (
```
Comment (forresttanaka) [inline components/common-data-items.js:154]: Just do `{item.starting_amount && (`. `truthyOrZero` is purely for numeric values that have a displayable zero value (the comment for the function describes what purpose it serves). `item.starting_amount` appears to be an embedded object, not a numeric value.

**Comment by forresttanaka** [comment]
Reviewing…

File: components/modification-table.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        columns={modificationsColumns}
+        keyProp="@id"
+      />
+    </DataGridContainer>
+  );
+}
+
+ModificationsTable.propTypes = {
+  // Modifications to display
+  modifications: PropTypes.arrayOf(PropTypes.object),
```
Comment (forresttanaka) [inline components/modification-table.js:47]: You can add `.isRequired` to this. SortableGrid needs to have data, and you check for its existence before passing it to this component anyway.

**Review by forresttanaka — CHANGES_REQUESTED** [review]

File: components/modification-table.js (hunk 2 above)
Comment (khzlin) [reply components/modification-table.js:47]: done

File: components/common-data-items.js (hunk 1 above)
Comment (khzlin) [reply components/common-data-items.js:154]: done

File: components/modification-table.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    title: "Fused Domain",
+  },
+  {
+    id: "tagged_protein",
+    title: "Tagged Protein",
+  },
+];
+
+/**
+ * Display a sortable table of the given treatments.
```
[the remark this answers is not in the document]
Comment (khzlin) [reply components/modification-table.js:31]: oh these are copy&paste errors, i'll change it to modifications

**Review by forresttanaka — DISMISSED** [review]
🆒

**Review by forresttanaka — APPROVED** [review]
🆒

**Pull request merged**